index.html

```diff
@@ -1016,6 +1016,12 @@ <h3>Endpoints</h3>
   <td><a>Take Element Screenshot</a></td>
  </tr>
 
+ <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/screenshot/full</td>
+  <td><a>Take Full Page Screenshot</a></td>
```

Comment:
We always capture the page, so maybe this should just be `Take Full Screenshot`?

Reply:
Done.